polaris-react/src/components/TextField/TextField.scss

```diff
@@ -17,11 +17,7 @@ $spinner-icon-size: 12px;
   cursor: text;
 
   svg {
-    fill: var(--p-color-icon);
-
-    #{$se23} & {
-      fill: var(--p-color-icon-subdued);
```

Comment:
This was previously incorrectly overriding the svg fill value on `disabled`.  Hence the storybook change. 
See [Figma](https://www.figma.com/file/jLLkmo9r28hiqPvf4s90E4/Polaris-Gen-3-Components-%5Balpha%5D?type=design&node-id=52147-22449&mode=design&t=qxZur6Fk6EpaoWVt-4)